src/lib/core/fiber_channel.h

```diff
@@ -137,7 +137,15 @@ struct fiber_channel {
 	struct rlist waiters;
```

Comment:
Please, adjust the comment above. Quoted relevant parts:

> A channel, once created is "open". I.e. anyone can read or
> write to/from a channel. A channel can be closed at any time,
> in which case, all messages currently buffered in a channel
> are destroyed, waiting readers or writers awoken with an error.

> Sending a message to a closed channel, as well as reading
> a message from such channel, always fails.

Reply:
Done.